schedule/migration/autoyast_switch_version_pre@pvm.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
I don't get the name of the file.

Reply:
It was temporary, I will not needa new schedule in the end, because setting the +VERSION: '15-SP4' works